modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -636,26 +661,114 @@ k8s.io/klog.Fatalf(...)
 main.main()
 	/remote-source/app/cnf-tests/pod-utils/hwlatdetect-runner/main.go:51 +0x897
 ----
-<1> The docker arguments provided by the user.
-<2> The latency threshold configured by the user using the `MAX_LATENCY` or the `HWLATDETECT_MAX_LATENCY` environment variables.
+<1> The `podman` arguments provided by the user.
```

Comment:
Do we update other sections to use podman rather than docker see further up in the same sections?

Reply:
@kquinn1204 The scope of the story is only the latency testing.

Reply:
I'll bring it up after the release, because there are other issues/questions as well.